spec/features/consumer/shopping/shopping_spec.rb

```diff
@@ -63,8 +64,10 @@
 
         it "shows a select with all order cycles, but doesn't show the products by default" do
           visit shop_path
+
           expect(page).to have_selector "option", text: 'frogs'
           expect(page).to have_selector "option", text: 'turtles'
+          expect(page).to have_content "choose when you want your order"
```

Comment:
shouldn't we use the I18n key to make this a bit less brittle?

Reply:
yeah, but there was some problem with the html translation I couldnt fix...